src/google/protobuf/compiler/java/java_map_field.cc

```diff
@@ -354,6 +356,7 @@ GenerateBuilderMembers(io::Printer* printer) const {
                  "public Builder clear$capitalized_name$() {\n"
                  "  internalGetMutable$capitalized_name$().getMutableMap()\n"
                  "      .clear();\n"
+                 "// @@protoc_insertion_point(builder_field_modifier_scope_after:$containing_type_full_name$.clear$capitalized_name$)\n"
```

Comment:
How about a simpler name:
@@protoc_insertion_point(builder_field_clear:$full_name$)
?
where full_name is descriptor->full_name().

Reply:
Sorry for the delay, but it is now done in trhe latest pull request